jsonapi.go

```diff
@@ -84,6 +85,7 @@ type Link struct {
 	Last     string `json:"last,omitempty"`
 	Next     string `json:"next,omitempty"`
 	Previous string `json:"previous,omitempty"`
```

Comment:
I'd recommend placing the comment as close to the field as possible, it might be more readable this way. 

```suggestion
        // Previous is kept for backwards compatibility. It is recommended to use the Prev field. 
	Previous string `json:"previous,omitempty"`
```

Provided you agree, let me know when you make the change and then I'll hit the merge button. :+1:

Reply:
Moved the comment !